cli/cmd/install_test.go

```diff
@@ -335,6 +337,38 @@ func testInstallOptions() (*installOptions, error) {
 	return o, nil
 }
 
+func testInstallValues() (*linkerd2.Values, error) {
```

Comment:
This is a helper function used with tests, to fetch full `Values` struct with Cert Key pairs from `/testdata`